.github/workflows/pytest.yml

```diff
@@ -198,36 +198,22 @@ jobs:
           path: coverage
       - name: List downloaded files
         run: |
-          ls -R
+          find coverage -type f
       - name: Upload test coverage report to CodeCov
         uses: codecov/codecov-action@v4
         with:
           directory: coverage
-
-  ci-notify:
-    runs-on: ubuntu-latest
-    if: ${{ always() }}
```

Comment:
This notification has been uninformative for a long time and creates a bunch of Slack noise. I approved its removal in once of Jan's PRs and have finally gotten fed up enough with it to cut it here...